GitStacked/WebContent/WorkoutDetails.jsp

```diff
@@ -8,14 +8,18 @@
 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
 <title>Insert title here</title>
 </head>
-<body>
-<table>
-<tr>
-<th></th>
-</tr>
-<tr>
-<td></td>
-</tr>
-</table>
-</body>
```

Comment:
HTML is hierarchical, for clarity and maintainability you want to indent it correctly.